engine/api/integration/artifact_manager/interface.go

```diff
@@ -15,11 +16,18 @@ import (
 // mockgen -source=interface.go -package mock_artifact_manager -destination=mock_artifact_manager/interface_mock.go ArtifactManager
 type ArtifactManager interface {
```

Comment:
I think it's a bit strange to import api package from plugin.
Maybe it's time to move this in the sdk